assets/js/scenes/department/ExpenditurePhaseSection/index.html

```diff
@@ -71,6 +71,10 @@ <h3 class="Title Title--section">Budgeted and actual expenditure comparison</h3>
                 <div class="ExpenditurePhaseSection-labelItem">
                     {% include 'components/ChartLabel/index.html' with index="3" label="Audited outcome" %}
                 </div>
+
+                <div class="ExpenditurePhaseSection-labelItem">
+                    {% include 'components/ChartLabel/index.html' with index="4" label="Actual expenditure" %}
+                </div>
```

Comment:
could you show this only if sphere==provincial?

Reply:
done